data/json/items/ammo/40.json

```diff
@@ -44,10 +44,10 @@
     "ammo_type": "40",
     "casing": "40_casing",
     "range": 14,
-    "damage": 28,
-    "pierce": 2,
+    "damage": 26,
```

Comment:
Could you update GAME_BALANCE.md to reflect this, or add a note describing why the damage rating changed? GAME_BALANCE cartridge looks like it's based on the 135 gr Federal Premium JHP Reduced Recoil cartridge.